.github/workflows/kuksa_databroker_build.yml

```diff
@@ -77,6 +81,8 @@ jobs:
       - name: Generate code coverage
         run: cargo llvm-cov --all-features --workspace --lcov --output-path lcov.info
       - name: Upload coverage to Codecov
+        # To use v4 a token must be specifed by "token: ${{ secrets.CODECOV_TOKEN }}""
```

Comment:
We get a warning "Node.js 16 actions are deprecated. " if using v3 (but it works).
If using v4 (to resolve warning) it seems  we need to create a token and upload.
Could possibly be done similar to how we do for other secrets at https://github.com/eclipse-kuksa/.eclipsefdn/blob/main/otterdog/eclipse-kuksa.jsonnet#L29